### ISSUE TITLE
Fix missing images bug.

### ISSUE THREAD

**Issue by gbmhunter** [gbmhunter/blog#10]
All images seem to go missing from site after upgrading `hugo`.

Run `brew unlink hugo` before running any `install` command.

Can reinstall version with `brew switch hugo 0.60.1`.

Find commits to point to on https://github.com/Homebrew/homebrew-core/search?p=1&q=hugo&type=Commits

**v0.57.0**

Images present.

```
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/cf67d08dd1a6201be82ac93feeabe2fe834f50ab/Formula/hugo.rb
```

**v0.58.0**

Images present.

```
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/1cf4fd682fbfe9e2d7b275e1e916b805c614f3df/Formula/hugo.rb
```

**v0.58.3**

Images present.

```
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/1eb598bac6dbd3e9234e50d7ab208074f5fcaac8/Formula/hugo.rb
```

**v0.59.0**

Images present.

```
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/6feb9c77f8b803409d31cbe633201fe3a1b15cd4/Formula/hugo.rb
```

**v0.59.1**

Images present.

```
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/6276d574f3efb2f690789d00ba987259fbfbd920/Formula/hugo.rb
```

**v0.60.1**

Images missing.

```
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/45012841e1c2da0bd680ff30a1bd6759fbfe1177/Formula/hugo.rb
```

**Comment by gbmhunter** [comment]
Images disappear on upgrade from hugo `v0.58.3` to `v0.59.0`.

Excerpt from `v0.59.0` release notes:

```
It is now possible to set the target format and the background fill color when processing images, e.g.:

{{ $image.Resize "600x jpg #b31280" }}
```

**Comment by gbmhunter** [comment]
Fixed in 5ea57260a334809cf2966dda90b17d02a8a4d9ed.
